edbdeploy/data/vmware-wkstn/r8-EDB-RA-1

```diff
@@ -0,0 +1,41 @@
+{
+  "pem1": {
+    "box": "rockylinux/8",
+    "box_version": "4.0.0",
+    "name": "pem1",
+    "provider": "vmware",
+    "shared_folders": [
+      {
+        "host_path": ".",
+        "share_name": "mech"
+      }
+    ],
+    "url": "https://app.vagrantup.com/rockylinux/boxes/8/versions/4.0.0/providers/virtualbox.box"
+  },
+  "barman1": {
+    "box": "rockylinux/8",
+    "box_version": "4.0.0",
+    "name": "barman1",
+    "provider": "vmware",
+    "shared_folders": [
+      {
+        "host_path": ".",
+        "share_name": "mech"
+      }
+    ],
+    "url": "https://app.vagrantup.com/rockylinux/boxes/8/versions/4.0.0/providers/virtualbox.box"
+  },
+  "primary1": {
+    "box": "rockylinux/8",
+    "box_version": "4.0.0",
+    "name": "primary1",
+    "provider": "vmware",
+    "shared_folders": [
+      {
+        "host_path": ".",
+        "share_name": "mech"
+      }
+    ],
+    "url": "https://app.vagrantup.com/rockylinux/boxes/8/versions/4.0.0/providers/virtualbox.box"
+  }
+}
```

Comment:
The Official Rockylinux does not support VMWare when it comes to Vagrant Boxes. I found another image that does. This is the reformatted file.

{
  "pem1": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "pem1",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "barman1": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "barman1",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "primary1": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "primary1",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  }
}